### PR TITLE
chore: add node v18 to test workflow

### PR DESCRIPTION
**Pull request by amareshsm**
| Q                 | A
| ----------------- | ---
| Bug fix?          | no
| Breaking change?  | no
| New feature?      | no
| Deprecations?     | no
| Tests added?      | no
| Docs updated?     | no
| Fixed tickets     | comma-separated list of tickets fixed by the PR, if any
| License           | MIT

Change: Add node v18 to test workflow.

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -16,7 +16,7 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [12.x, 14.x, 16.x, 17.x]
+        node-version: [12.x, 14.x, 16.x, 18.x]
         webpack-version: [default, ^4.0.0]
         exclude:
           - node-version: 17.x
```
Comment (danez) [inline .github/workflows/build.yml:22]: webpack 4 does not work on node 17+. Can you please add a new exclude here?

Comment (danez) [inline .github/workflows/build.yml:22]: webpack 4 does not work on node 17+. Can you pleasechange the exclude here for node 18?
```suggestion
          - node-version: 18.x
```

**Review by danez — CHANGES_REQUESTED** [review]
Thanks, So far so good, only the exclude needs to change too. See inline comment.

File: .github/workflows/build.yml (hunk 1 above)
Comment (amareshsm) [reply .github/workflows/build.yml]: done excluded node 18.

**Review by danez — APPROVED** [review]

**Pull request merged**